api/v1alpha1/oadp_types.go

```diff
@@ -160,6 +160,12 @@ type NodeAgentConfig struct {
 	// +kubebuilder:validation:Enum=restic;kopia
 	// +kubebuilder:validation:Required
 	UploaderType string `json:"uploaderType"`
+
+	// UnsupportedNodeAgentArgs accepts a string of server args and their values to be supplied to nodeagent daemon set
+	// each pair of server arg and value is comma separated if there are multiple args to be supplied
+	// This will replace and override any other args specified via DPA for the nodeagent
```

Comment:
today, we do not pass any arg to node-agents containers. But if this is something that will be done in the future, would change code, as it only appends values, not clean them previously

Reply:
The future PR should take care of it, right ?

Reply:
would remove the comment if we do not change that in code now

Reply:
modified